src/controller/.routes.yml

```diff
@@ -51,48 +47,48 @@ rebalance:
 
     # V3 pools with multiple strategies
     - schedule:
-        rate: cron(40 3-6,12-18 ? * * *)
+        rate: cron(40 3-6,14-18 ? * * *)
         input:
           pools: "vUSDT"
 
     - schedule:
-        rate: cron(35 3-6,12-18 ? * * *)
+        rate: cron(35 3-6,14-18 ? * * *)
         input:
           pools: "vUNI"
 
-    - schedule:
-        rate: cron(30 3-6,12-18 ? * * *)
-        input:
-          pools: "vaDAI"
+    # - schedule:
+    #     rate: cron(30 3-6,14-18 ? * * *)
+    #     input:
+    #       pools: "vaDAI"
```

Comment:
Better to delete it. 2 more commented schedules down in this file

Reply:
Aggressive pools ops are manually handled for now. Will be enabled again so commenting for now.